### PR TITLE
Add info about database snapshots and #blockchain-dev for support

### PR DESCRIPTION
**Pull request by jamiew**
- Discord seems preferred for support (right?)
- there are DB backups now but you should be careful about them
- thanks for all you do, marc

### CODE REVIEW

**Comment by PharkMillups** [comment]
Looks good to me but I'll wait for one of the committers to chime in before merging.

**Review by madninja — APPROVED** [review]

**Pull request merged**
